backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -77,6 +77,12 @@ fun Route.configureShorkInterpreterControllerV1() {
         return@get
     }
 
+    get("/lobby") {
+        val allLobbies = shorkUseCase.getAllLobbies()
+        call.respond(HttpStatusCode.OK, mapOf("lobbies" to allLobbies))
```

Comment:
you _could_ introduce a DTO here if you wanted to do it even cleaner. But we have not done that so far, so also part of  another story I guess 🤷🏻